docs/x509/reference.rst

```diff
@@ -2854,6 +2878,15 @@ instances. The following common OIDs are available as constants.
 
         Corresponds to the dotted string ``"2.5.29.24"``.
 
+
+.. class:: OCSPExtensionOID
```

Comment:
Should we differentiate between `OCSPRequestExtension` and `OCSPResponseExtension`?

Reply:
Nonce, for example, is the same extension in both response and request right down to the OID so I think differentiating is probably undesirable.